swagger.json

```diff
@@ -5548,6 +5773,59 @@
             "in": "query",
             "type": "string"
         },
+        "addressAddParam": {
+            "name": "addresses",
```

Comment:
Should `addresses` be plural?

Reply:
It looks like we allow multiple addresses in a single `POST` call and the API expects a plural arg.